lib/ansible/module_utils/remote_management/dellemc/dellemc_idrac.py

```diff
@@ -58,7 +58,7 @@ def connect(self):
             self.module.fail_json(**results)
 
         # Connect to iDRAC
-        if idrac_ip == '' or idrac_user == '' or idrac_pwd == '':
+        if not all((idrac_ip, idrac_user, idrac_pwd)):
```

Comment:
I'd still prefer this check to be moved to the initializer

Reply:
moved this validation to initializer section